tests/admin_views/models.py

```diff
@@ -335,6 +335,14 @@ class Vodcast(Media):
     released = models.BooleanField(default=False)
 
 
+class Screencast(Media):
+    media = models.OneToOneField(Media, models.CASCADE, primary_key=True, related_name='screencast')
+    title = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.title
```

Comment:
You can reuse the `PlotDetails` model instead of defining a new one as it slow downs the test suite execution.